Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -1518,6 +1519,24 @@ static bool exceedsRenderTreeSizeSizeThreshold(uint64_t thresholdSize, uint64_t
     }, webPageIDInMainFrameProcess());
 }
 
+String WebPageProxy::presentingApplicationBundleIdentifier() const
+{
+    if (std::optional auditToken = presentingApplicationAuditToken()) {
```

Comment:
Will `auto auditToken = ...` now work here?

Reply:
Yes, but my preference is to be more explicit so it's obvious this is an optional type.